package.json

```diff
@@ -36,23 +49,24 @@
     "eslint-plugin-jest": "^29.0.0",
     "eslint-plugin-react": "^7.35.0",
     "eslint-plugin-react-hooks": "^5.0.0",
-    "globals": "^16.0.0",
-    "typescript-eslint": "^8.6.0"
+    "globals": "^16.0.0"
   },
   "devDependencies": {
     "@changesets/cli": "^2.27.7",
     "@changesets/get-github-info": "^0.6.0",
     "@tsconfig/node22": "^22.0.2",
-    "eslint": "^9.8.0",
+    "eslint": "^9.22.0",
+    "jiti": "^2.6.1",
```

Comment:
`jiti` enables TypeScript eslint config support, which seemed like a simple way to test the source code directly.